presto-ml/src/test/java/io/prestosql/plugin/ml/TestLearnAggregations.java

```diff
@@ -51,18 +50,14 @@
 
 public class TestLearnAggregations
 {
-    private static final TypeManager typeManager;
+    private static final Metadata METADATA = createTestMetadataManager();
+    private static final TypeManager typeManager = METADATA.getTypeManager();
```

Comment:
nit: why is one field in CAPS and other is not?

Reply:
The field was preexisting.  I'll fix it